acme/account_test.go

```diff
@@ -66,6 +66,23 @@ func TestKeyToID(t *testing.T) {
 	}
 }
 
+func TestAccount_GetLocation(t *testing.T) {
+	locationPrefix := "https://test.ca.smallstep.com/acme/foo/account/"
```

Comment:
To properly test that the URL is the right one you should get an account for NewAccount, or at least use something like:
```go
locationPrefix := getAccountLocationPath(ctx, linker, "")
```